src/main/java/org/apache/maven/shared/utils/logging/MessageUtils.java

```diff
@@ -201,4 +201,21 @@ public void run()
             Runtime.getRuntime().addShutdownHook( shutdownHook );
         }
     }
+
+    /**
+     * Get the terminal width or 0 if the width can not be determined.
+     *
+     * @return the terminal width
+     */
+    public static int getTerminalWidth()
+    {
+        if ( JANSI )
+        {
+            return AnsiConsole.getTerminalWidth();
+        }
+        else
+        {
+            return 0;
```

Comment:
-1 is a more common "I couldn't figure out the correct value" flag in these sorts of methods.

Reply:
Up to you, jansi returns 0 in that case, but I can check and translate it to -1.

Reply:
Fixed.  The related maven PR does not have to be changed since the test is `<= 0`
https://github.com/apache/maven/pull/434/commits/1cf1487af8081faa55eb8bd9956c8f182f76b4aa#diff-ff39a40ae9303293688fbd2f79ad043c73816772bbd153e51681fdb8c384d085R174-R175